include/seqan3/search/configuration/all.hpp

```diff
@@ -128,4 +128,12 @@
  * The following example demonstrates the usage of the dynamic configuration:
  *
  * \include test/snippet/search/dynamic_hit_configuration_example.cpp
+ *
+ * \subsection search_configuration_subsection_parallel 6: Parallel Configuration
+ *
+ * This configuration determines the maximal number of threads the search algorithm can use.
+ *
+ * The seqan3::search_cfg::parallel configuration element can be combined with any other search configuration.
+ *
+ * \include test/snippet/search/configuration_parallel.cpp
```

Comment:
Can you add an example to the snippet of how one can use the thread_count member?